### ISSUE TITLE
[9.2.1] Doubled computer entries with OS name column

### ISSUE THREAD

**Issue by pVesian** [glpi-project/glpi#3566]
Hi,

On 9.2.1 a client has the following problem:
![screenshot-192 168 19 158-2018-02-15-17-21-21](https://user-images.githubusercontent.com/3509962/36267642-b5205910-1274-11e8-8c5b-68755cd1dd68.png)

The computer appears in double! Both entries have the same ID. This problem disappears when the "OS name" column is removed from the view.
I've identified the query that causes this:
```sql
SELECT
    'glpi' AS currentuser,
    `glpi_computers`.`entities_id`,
    `glpi_computers`.`is_recursive`,
    `glpi_computers`.`name` AS `ITEM_0`,
    `glpi_computers`.`id` AS `ITEM_0_id`,

    `glpi_users`.`name` AS `ITEM_1`,
    `glpi_users`.`realname` AS `ITEM_1_realname`,
    `glpi_users`.`id` AS `ITEM_1_id`,
    `glpi_users`.`firstname` AS `ITEM_1_firstname`,
    `glpi_locations`.`completename` AS `ITEM_2`,
    `glpi_operatingsystems_9719987b154aaf3b42c3db32aef59090`.`name` AS `ITEM_3`,

    `glpi_computertypes`.`name` AS `ITEM_4`,
    `glpi_computermodels`.`name` AS `ITEM_5`,
    `glpi_states`.`completename` AS `ITEM_6`,
    `glpi_groups`.`completename` AS `ITEM_7`,
    `glpi_computers`.`serial` AS `ITEM_8`,
    `glpi_computers`.`id` AS id

FROM `glpi_computers`

LEFT JOIN `glpi_users` ON (`glpi_computers`.`users_id` = `glpi_users`.`id` )
LEFT JOIN `glpi_locations` ON (`glpi_computers`.`locations_id` = `glpi_locations`.`id` )

LEFT JOIN `glpi_items_operatingsystems` ON (
    `glpi_computers`.`id` = `glpi_items_operatingsystems`.`items_id`
    AND `glpi_items_operatingsystems`.`itemtype` = 'Computer' )
LEFT JOIN `glpi_operatingsystems` AS `glpi_operatingsystems_9719987b154aaf3b42c3db32aef59090` ON (
    `glpi_items_operatingsystems`.`operatingsystems_id` = `glpi_operatingsystems_9719987b154aaf3b42c3db32aef59090`.`id` )

LEFT JOIN `glpi_computertypes` ON (`glpi_computers`.`computertypes_id` = `glpi_computertypes`.`id` )
LEFT JOIN `glpi_computermodels` ON (`glpi_computers`.`computermodels_id` = `glpi_computermodels`.`id` )
LEFT JOIN `glpi_states` ON (`glpi_computers`.`states_id` = `glpi_states`.`id` )
LEFT JOIN `glpi_groups` ON (`glpi_computers`.`groups_id` = `glpi_groups`.`id` )


WHERE `glpi_computers`.`is_deleted` = '0' AND `glpi_computers`.`is_template` = '0'
ORDER BY ITEM_0 ASC
LIMIT 0, 200```

The cause comes from these parts:
```sql
`glpi_operatingsystems_9719987b154aaf3b42c3db32aef59090`.`name` AS `ITEM_3`
```

More precisely:
```sql
LEFT JOIN `glpi_items_operatingsystems` ON (
    `glpi_computers`.`id` = `glpi_items_operatingsystems`.`items_id`
    AND `glpi_items_operatingsystems`.`itemtype` = 'Computer' )
LEFT JOIN `glpi_operatingsystems` AS `glpi_operatingsystems_9719987b154aaf3b42c3db32aef59090` ON (
    `glpi_items_operatingsystems`.`operatingsystems_id` = `glpi_operatingsystems_9719987b154aaf3b42c3db32aef59090`.`id` )
```

Thanks.

**Comment by trasher** [comment]
Could you check if there is only one operating system attached to this computer?

**Comment by ladenree76000** [comment]
looks like this  
https://github.com/fusioninventory/fusioninventory-for-glpi/issues/2376

**Comment by pVesian** [comment]
Sorry, I didn't catch the notification.
@trasher: yes, there are 2 OS, here's a screenshot:
![screenshot-192 168 19 158-2018-03-16-19-33-17](https://user-images.githubusercontent.com/3509962/37538559-421148fa-2951-11e8-9fad-9297f0a03fa6.png)
@ladenree76000: yes, looks the same. So it's a fusioninventory problem?

**Comment by trasher** [comment]
Yes it is. It's been fixed in 9.2+2.0RC1 (see linked ticket; I gave a query to clean duplicates, but did not have feedback).
